Atomic/backends/_containers_storage.py

```diff
@@ -250,7 +250,10 @@ def start_container(self, name):
         raise UnderDevelopment()
 
     def stop_container(self, con_obj, **kwargs):
-        raise UnderDevelopment()
+        # If we plant to honor stop labels here, we will
```

Comment:
nit, "plant to" to  "plan to" or just drop both words and go to "If we honor stop..."